core/modules/ccontrol/queryMsg.cc

```diff
@@ -43,19 +43,23 @@ int queryMsgGetCount(int session) {
 }
 
 // Python call: msg, chunkId, code, timestamp = queryMsgGetMsg(session, idx)
-std::string queryMsgGetMsg(int session, int idx, int* chunkId, int* code, time_t* timestamp) {
+std::string queryMsgGetMsg(int session, int idx, int* chunkId, int* code, std::string* severity, time_t* timestamp) {
```

Comment:
I'd suggest to use references (&) instead of pointers (*) for both severity and timestamp.